.golangci.yml

```diff
@@ -162,6 +162,9 @@ issues:
     - linters:
         - gosec
       text: "G304: Potential file inclusion via variable"
+    - linters:
+        - gosec
+      text: "G115: integer overflow conversion"
```

Comment:
I would prefer not globally ignoring this finding. Let's plese revert the change here and take a look at the occurences, verify that it's fine to ignore there and then add //nolint comments if appropriate

Reply:
I think the one in admission is fine, the one in zap/flags.go could probably use some additional validation